apps/site/lib/site_web/templates/fare/_nearby_locations.html.eex

```diff
@@ -1,16 +1,55 @@
-<div class="fare-sales-locations">
-  <h3>Nearby MBTA fare retail sales locations</h3>
-  <div class="c-location-cards c-location-cards--background-gray small-set grouped">
-    <%= for {%Fares.RetailLocations.Location{name: name, address: address} = location, distance} <- @fare_sales_locations do %>
-      <%= link to: GoogleMaps.direction_map_url(@search_position, location), class: "c-location-card c-location-card--flex", target: "_blank" do %>
-        <div class="c-location-card__name">
-          <%= name %>
+<div class="c-location-cards">
+  <%= for {location, distance} <- @locations do %>
+    <%
+      name = Map.get(location, :name)
+      address = Map.get(location, :address)
+      retail_fvm = Map.get(location, :retail_fvm)
+      stop_id = Map.get(location, :stop_id)
+    %>
+    <div class="c-location-card c-location-card--rsl">
+      <div class="c-flex-space-between">
+        <div class="c-location-card__place">
+          <%= if stop_id do %>
+            <%= link name, to: stop_path(@conn, :show, stop_id) %>
+          <% else %>
+            <%= name %>
+          <% end %>
         </div>
         <div class="c-location-card__distance"><%= round_distance(distance) %></div>
-        <div class="c-location-card__description">
+      </div>
+      <%= if address do %>
+        <div class="c-location-card__location">
           <%= address %>
         </div>
       <% end %>
-    <% end %>
-  </div>
+      <%= if retail_fvm do %>
+        <div class="c-location-card__type">
+          <%= retail_fvm %>
+        </div>
+      <% end %>
+
+      <div class="dropdown show c-location-card__directions">
+        <a href="#" role="button" id="get-directions-dropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

Comment:
Since one of these gets generated for every location, we'll want a unique `id` here 
(and then should adjust the corresponding CSS to not use the id as the selector: `.c-location-card__directions [data-toggle="dropdown"]` would be a-ok selector to me)

Reply:
The only place this id is used is in the "aria-labelledby" of the dropdown menu content itself.  I did a little searching, but I'm not totally sure what the best use of aria labelling is.  Do we need this id?  How do I properly set aria labels on the dropdown?

Reply:
Yeah, I think it's good to keep the IDs for this! 
You can (and should!) still use them here for the `aria-labelledby` attribute; I'm suggesting you give the ID a unique name (instead of "get-directions-dropdown" for every single location)

Reply:
What does aria-labelledby use to make the label?  (Changing the id right now!)

Reply:
You'll have to update the value of `aria-labelledby` to match the ID you're associating it with, I think that answers your question?